### PR TITLE
Added some questions and answers to the front page of the folkrnn app

### PR DESCRIPTION
**Pull request by boblsturm**

### CODE REVIEW

File: folk_rnn_site/composer/templates/composer/_header.html [hunk 1]
```diff
@@ -27,7 +27,9 @@ <h2 class="content-subhead">What is ABC Notation?</h2>
             <li><a href="http://abcnotation.com/blog/2010/01/31/how-to-understand-abc-the-basics/">Understanding ABC</a></li>
             <li><a href="http://abcnotation.com/wiki/abc:standard:v2.1">ABC format standard</a>
         </ul>
+        <h2 class="content-subhead">What's the difference between the models?</h2>
+        <p>In the controls on the left, you can select between two models built using data from <a href="http://thesession.org">thesession.org</a>: (w/ :||:) and (w/o  :||:). The first model is trained on abc with the repeat signs preserved. The second is trained on abc with the repeats made explicit.</p>
         <h2 class="content-subhead">Are there issues with playback on this site?</h2>
```
Comment (tobyspark) [inline folk_rnn_site/composer/templates/composer/_header.html]: Took me a moment to realise that 'made explicit' wasn't a mistake. The programmer term would be 'loops unrolled'. Is there a better example or clearer way of saying this?

**Comment by boblsturm** [comment]
"Repeats unrolled" is fine. :)

**Comment by tobyspark** [comment]
If you add that as a commit, I'll merge the pull request.

**Comment by tobyspark** [comment]
You did everything perfectly to create the pull request. We're now improving it. So tweak the text to include 'unrolled', commit, push and this pull request will update itself. And then I hit the 'merge' button.

All a bit of a faff over a single bit of prose, but hopefully you can see the value for chunkier (and potentially more dangerous) changes, so it's a good rehearsal =]

**Comment by tobyspark** [comment]
Closed? I haven't merged it, waiting for the 'unrolled' commit.

**Comment by boblsturm** [comment]
I closed it because I made another change... but now I see that wasn't the way to do it.

**Comment by boblsturm** [comment]
I made another pull request though.

**Comment by tobyspark** [comment]
I don't see another pull request. If you just commit, push, it should appear here. But, having said that, if you have committed and it doesn't appear here... I'm not sure.

**Comment by boblsturm** [comment]
I just pushed.

**Comment by boblsturm** [comment]
How about now?

**Pull request merged**